src/cleanvision/issue_managers/image_property.py

```diff
@@ -207,7 +207,7 @@ def get_image_mode(image: Image) -> str:
         imarr = np.asarray(image)
         if len(imarr.shape) == 2 or (
             len(imarr.shape) == 3
-            and (np.diff(imarr.reshape(-1, 3).T, axis=0) == 0).all()  # type: ignore
+            and (np.diff(imarr.reshape(-1, 3).T, axis=0) == 0).all()
```

Comment:
Does this pass mypy typecheck? np.diff for my local versions caused error as an untyped function (thats why `# type: ignore` was there)

Reply:
It did pass for me on local and looks like it's passing on CI as well